travis/Dockerfile-travis-watch-build

```diff
@@ -0,0 +1,29 @@
+FROM node:6.2
+MAINTAINER ODL DevOps <mitx-devops@mit.edu>
+
+# this dockerfile builds the hub image for the watch container
+# we don't use this directly, instead we push to docker-hub, and
+# then Dockerfile-travis-watch uses that pushed image to bootstrap
+# itself
+
+RUN apt-get update && apt-get install libelf1
+
+RUN mkdir /src
+
+WORKDIR /src
+
+RUN adduser --disabled-password --gecos "" mitodl
+
+RUN chown -R mitodl:mitodl /src
+
+USER mitodl
+
+COPY package.json /src
```

Comment:
What do you think of combining these multiple `COPY` statements into one (e.g. `COPY package.json npm-shrinkwrap.json ./webpack_if_prod.sh /src/`)? It would be a very minor reduction to the layers Travis would have to pull. Tradeoff is probably that's a usage that I haven't seen in other ODL Dockerfiles so it might be unfamiliar to some.